_data/sources/meetup.js

```diff
@@ -1,17 +1,63 @@
-const fetch = require("node-fetch"),
+const {GraphQLClient, gql} = require("graphql-request"),
     group = require("./templates/group"),
     event = require("./templates/event");
 
 class meetup {
-    constructor() {
+    constructor(token = process.env.MEETUP_TOKEN) {
         this.groups = require("./groupIds/meetup").filter((value, index, self) => self.indexOf(value) === index);
-        this.apiEvents = (group) => 'https://api.meetup.com/' + group + '/events';
+        this.graphQLClient = new GraphQLClient('https://api.meetup.com/gql', {
+            headers: {
+                authorization: 'Bearer ' + token,
+                method: 'POST',
+                dataType: 'jsonp'
+            },
+        });
+        this.apiEvents = (group) => gql`
+            {
+                groupByUrlname(urlname: ${group}) {
+                    unifiedEvents {
+                        edges {
+                            node {
+                                title
+                                shortUrl
+                                venue
+                                description
+                                dateTime
+                                endTime
+                                going
+                                maxTickets
+                                fees
+                                price
+                                groupName
+                                groupLink
+                                source
```

Comment:
This is not a valid parameter for the Meetup API, neither REST nor GraphQL, in this case `source` comes after the API call (not part of it) and is populated by the string "Meetup" (as can be seen on line 64).

Reply:
should be fixed by c19c0ba